Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,29 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const result =
+    // step 1.2.4
+    Math.floor(
+      100 *
+        tasks
+          // step 1.2.1
+          .map(x => x.duration / 60)
+          // step 1.2.2
+          .filter(x => x >= 2)
+          // step 1.2.3
+          .map(x => x * hourlyRate)
+          .reduce((accumulator, number) => accumulator + number)
```

Comment:
The `number` here is an `amount`. And I would choose `total` for the accumulator, because you are computing a running total. I have made it a habit to always explicitly add the optional, second argument. 

```js
.reduce((total, amount) => total + amount, 0)
```